packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -14,18 +14,41 @@ import {
   TabBarSection,
   FlexCenter,
   Report,
-  TabBar,
+  StickyTabBar,
   Tabs,
   Tab,
   TabPanel,
   YearSelector,
 } from '../components';
-import { DEFAULT_DASHBOARD_GROUP, DEFAULT_DATA_YEAR } from '../constants';
+import { DEFAULT_DASHBOARD_GROUP, DEFAULT_DATA_YEAR, NAVBAR_HEIGHT_INT } from '../constants';
 
 const DashboardSection = styled(FlexCenter)`
   min-height: 31rem;
 `;
 
+const ScrollToTopButtonContainer = styled.div`
+  position: sticky;
+  bottom: 29px;
+  padding-right: 32px;
+  cursor: pointer;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const ScrollToTopButtonImg = styled.img`
+  width: 40px;
+  height: 40px;
+`;
+
+const ScrollToTopButton = React.memo(({ onClick }) => (
+  <ScrollToTopButtonContainer onClick={onClick}>
+    <ScrollToTopButtonImg src="/images/up-arrow.svg" alt="back to top" />
```

Comment:
I would usually prefer to use the [material-ui icons](https://material-ui.com/components/material-icons/#material-icons) rather than adding a new image. We will just need to add a box around the icon to make it look the same as the design. An extra advantage of this is then you can control the icon on interactions (eg the color on hover)

Reply:
Good call - changed :-)